core/trino-main/src/main/java/io/trino/sql/planner/PlanOptimizers.java

```diff
@@ -855,6 +871,7 @@ public PlanOptimizers(
                             // DetermineTableScanNodePartitioning is needed to needs to ensure all table handles have proper partitioning determined
                             // Must run before AddExchanges
                             .add(new DetermineTableScanNodePartitioning(metadata, nodePartitioningManager, taskCountEstimator))
+                            .add(new MultipleDistinctAggregationToMarkDistinct())
```

Comment:
We had `MultipleDistinctAggregationToMarkDistinct` once and now we have twice. What's the reason?

Reply:
Does not make sense.